src/main/webapp/view/transcript/transcript-area.js

```diff
@@ -49,9 +56,28 @@ export default class TranscriptArea {
     url.searchParams.append(TranscriptArea.#PARAM_ID, window.LECTURE_ID);
     const transcriptResponse = await fetch(url);
     const transcriptLines = await transcriptResponse.json();
+    // No transcript lines are available for this lecture.
+    if (transcriptLines.length == 0) {
+      TranscriptArea.displayNoTranscriptMessage();
+      return;
+    }
     TranscriptArea.addTranscriptLinesToDom(transcriptLines);
   }
 
+  /**
+   * Displays a message in the transcript container if there is no
+   * transcript available for the lecture recording.
+   */
+  static displayNoTranscriptMessage() {
+    // TODO: Get the transcript container from a getter method once #286
+    // is merged.
+    const transcriptContainer =
+        document.getElementById(TranscriptArea.#TRANSCRIPT_CONTAINER);
+    transcriptContainer.innerText = TranscriptArea.#TRANSCRIPT_ERROR_MESSAGE;
+    transcriptContainer.classList.add('text-center');
+    TranscriptArea.#hasTranscript = false;
```

Comment:
This has nothing to do with displaying there is no transcript message. Seems like a scope creep.